MonstieWash/Assets/Scripts/Player/StuckItem.cs

```diff
@@ -65,6 +69,8 @@ public void SetStuck(bool stuck)
     /// <returns>Whether this wiggle unstuck the item.</returns>
     public bool Wiggle()
     {
+        m_pickingTask.TaskProgress = 100f*(m_initialWiggleCount - wiggleCount + 1)/m_initialWiggleCount;
```

Comment:
This is literally the only thing wrong, and it's my fault lol. I assume you got this formula by copying it from the `MaxRotation` getter but upon a second look it turns out I'm dumb and got the formula wrong and added a +1 for some reason. Here is what it looks like fixed. Pls also update the one in the getter tyty.
```suggestion
        m_pickingTask.TaskProgress = 100f*(m_initialWiggleCount - wiggleCount)/m_initialWiggleCount;
```